java/core/src/main/java/co/worklytics/psoxy/gateway/ConfigService.java

```diff
@@ -11,18 +11,55 @@ interface ConfigProperty extends Serializable {
 
         // shared? (across multiple instances?)
         // local? (per-instance secrets)
+
+        /**
+         * @return whether cached value for property must be revalidated with origin before re-use
+         *  (equivalent to HTTP `Cache-Control: no-cache` semantics as defined by RFC 7234)
+         *
+         * @see 'https://www.rfc-editor.org/rfc/rfc7234#section-5.2.2'
+         */
+        default Boolean noCache() {
+            return false;
+        }
```

Comment:
This is actually too conservative for the ACCESS_TOKEN use case. If we had memcache / redis caching at proxy- instance level (rather than lambda/cloud function instance level), we could use that just fine.  

The problem is caching at process/thread memory such that all lambda/cloud function instances don't see changes. 

so could drop reliance on standard definition of `no-cache` and call this `noLocalCache` to allow for future use of memcache/redis as caching solution in front of SSM ParameterStore / GCP Secret Manager / etc.

Reply:
Just personal taste and I know the RFC, but maybe clear if the property is just `cache` so when defining the properties in *ConfigService.ConfigProperty* implementations we don't need to use the double negation ( `REFRESH_ENDPOINT(false)` -> `REFRESH_ENDPOINT(true)` )